workflow-dev/manifests/deis-logger-svc.yaml

```diff
@@ -11,10 +11,5 @@ spec:
   - port: 80
     name: http
     targetPort: http
-  - port: 514
-    name: transport
-    targetPort: 1514
-    protocol: UDP
   selector:
     app: deis-logger
-    
```

Comment:
Should put the empty line back at the end of the file. Almost universally, we include a blank line at the end of files. Don't ask me why, but this is definitely an outlier.